pdpserver/server/server.go

```diff
@@ -249,6 +258,11 @@ func NewServer(opts ...Option) *Server {
 	}
 }
 
+// GRPCServer expose the underlying gRPC Server
+func (s *Server) GRPCServer() *grpc.Server {
+	return s.requests.proto
```

Comment:
There's a race that causes this to be `nil` sometimes, we need to wait on a non-nil server

Reply:
Looks like s.requests.proto is being recreated in LoadPolicies()

https://github.com/infobloxopen/themis/pull/245/files/32161ccd581a27f1984710426a54cf488aef7d92#diff-7e71dc72409d1564ddfa250aac5a9eabR556

Why can't the server be configured at Server creation time? What is changing during LoadPolicies that requires a NewGRPC server to be instantiated?

Reply:
it's not being re-created, it's just being created very late in `Serve()`. I'll see if I can get this moved to the appropriate place `NewServer()`